sklearn/feature_selection/_univariate_selection.py

```diff
@@ -254,6 +259,21 @@ def chi2(X, y):
     array([15.3...,  6.5       ,  8.9...])
     >>> p_values
     array([0.0004..., 0.0387..., 0.0116... ])
+
+    Using chi2 with continuous data (requires binning):
```

Comment:
Even if you could argue it is useful, I don't think it is worth adding a code example with binning, only mention it in the docstring above.

Reply:
Yes, I agree with you.